packages/workbox-cache-expiration/CacheExpirationPlugin.mjs

```diff
@@ -92,11 +92,64 @@ class CacheExpirationPlugin {
    *         fresh, or `null` if the `Response` is older than `maxAgeSeconds`.
    */
   cachedResponseWillBeUsed({cacheName, cachedResponse} = {}) {
-    if (this.isResponseFresh({cacheName, cachedResponse})) {
-      return cachedResponse;
+    let isFresh = this._isResponseDateFresh(cachedResponse);
+
+    // Expire entries to ensure that even if the expiration date has
+    // expired, it'll only be used once.
+    const cacheExpiration = this._getCacheExpiration(cacheName);
+    cacheExpiration.expireEntries();
```

Comment:
There's a bunch of lines of code changed, but as far as I can tell from the PR's description, this is the only one that actually matters—can you confirm that?

Reply:
Yeah this is the important bit.